borg/platform_windows.pyx

```diff
@@ -0,0 +1,295 @@
+from __future__ import absolute_import
```

Comment:
I get `AttributeError: module 'borg.platform' has no attribute 'node'` without it